README.adoc

```diff
@@ -22,5 +22,39 @@ image:img/screenshot.png[alt="screenshot"]
 * Multi-PCB feature (different PCB variants of the same schematic)
 * Automatic netlist synchronisation between schematic and board
 
-*You want to give LibrePCB a try?* +
+
+[#projectstatus]
+== Project Status
+
+LibrePCB is still under development and many things are not yet
+working as expected. So here an overview about the project status:
+
+=== What's working (more or less)
+
+* Downloading, creating and modifying libraries
+* Drawing simple schematics with multiple sheets
+* Creating *very* simple PCBs (incl. Gerber export)
+
+=== What's not working (or even missing)
+
+* Schematic editor:
+** No hierarchical sheets
+** No buses
+** No BOM export
+* Board editor:
+** No airwires
+** No texts (e.g. on silkscreen)
+** No planes (filled copper areas)
+** No 3D board viewer
+** *Generally very limited functionality and many bugs because
+   it's still provisional!*
```

Comment:
I would replace provisional with "work in progress" or "under development".

Reply:
done